README.md

```diff
@@ -1,7 +1,7 @@
 About pydicom
 =============
 
-Home: http://pydicom.readthedocs.org/
+Home: http://pydicom.github.io/pydicom
```

Comment:
Typically this would be fixed in the `meta.yaml` and then re-rendering would propagate the change through to the `README.md`.

Reply:
Great. so closing the issue then